### PR TITLE
improve test readme

### PR DESCRIPTION
**Pull request by janpio**
### Checklist
- [ ] [Reported an issue](http://cordova.apache.org/contribute/issues.html) in the JIRA database
- [ ] Commit message follows the format: "CB-3232: (android) Fix bug with resolving file paths", where CB-xxxx is the JIRA ID & "android" is the platform affected.
- [ ] Added automated test coverage as appropriate for this change.

### CODE REVIEW

File: test/README.md [hunk 1]
```diff
@@ -50,8 +50,10 @@ end-to-end, or instrumented, tests. Unit tests do not require any particular
 environment to run in, but the instrumented tests, however, require a connected
 Android device or emulator to run in.
 
-To run the unit tests, run: `gradlew test`.
-To run the instrumented tests, run: `gradlew connectedAndroidTest`.
+- To run the unit tests, run: `gradlew test`.
```
Comment (brodybits) [inline test/README.md]:
```suggestion
- Shell command to run the unit tests: `gradlew test`.
```

File: test/README.md [hunk 2]
```diff
@@ -50,8 +50,10 @@ end-to-end, or instrumented, tests. Unit tests do not require any particular
...
+- To run the instrumented tests, run: `gradlew connectedAndroidTest`.
```
Comment (brodybits) [inline test/README.md]:
```suggestion
- Shell command to run the instrumented tests: `gradlew connectedAndroidTest`.
```

**Comment by brodybits** [comment]
I wonder if we should add a pointer to apache/cordova#54?

File: test/README.md (hunk 1 above)
Comment (janpio) [reply test/README.md]: "Shell" is not a very common word in Windows land.
Besides, we never explicitly mention that those are command line commands or similar, do we?

**Comment by janpio** [comment]
Already done in https://github.com/apache/cordova/issues/54#issuecomment-441290852

**Comment by brodybits** [comment]
@janpio should we merge this?

**Pull request merged**